R/RLearner_regr_gbm.R

```diff
@@ -30,10 +32,39 @@ trainLearner.regr.gbm = function(.learner, .task, .subset, .weights = NULL, ...)
   f = getTaskFormula(.task)
   if (is.null(.weights)) {
     f = getTaskFormula(.task)
-    gbm::gbm(f, data = getTaskData(.task, .subset), ...)
+
+    params <- list(...)
+    if("alpha" %in% names(params)){
```

Comment:
Space after if.

Reply:
Done. Perhaps you can update the link on the PR message ? 

````
Please make sure you have read our coding guidelines:
https://github.com/mlr-org/mlr/wiki/mlr-Coding-Guidelines
````

and remplace with https://www.notion.so/Style-Guide-740bc663207a4bbb9a457987bda6fd91

Reply:
Thanks, done.